api/routes/users.routes.js

```diff
@@ -0,0 +1,11 @@
+module.exports = app => {
+    const users = require('../controllers/users.controller.js');
+
+    const router = require('express').Router();
+
+    router.post('/signup', users.signup);
+
+    router.post('/signin', users.signin);
+
+    app.use('/api', router);
+}
```

Comment:
It shouldn't be a function. Just make routes here and use `app.use` inside root file